### ISSUE TITLE
Opening Zazu loses focus of the current window

### ISSUE THREAD

**Issue by gbdematos** [bayleeadamoss/zazu#306]
The situation: 

I'm using Chrome, writing an e-mail. I want to do a quick math, so Alt+Space, type the operation, get the result, and, when pressing Enter to copy the result to clipboard, or pressing Alt+Space again or ESC to close Zazu, I'd like to return the focus to the window I was in (Chrome in this case) and just continue what I was doing. 

In the current behavior, after closing Zazu I have to click or Alt+Tab to get the focus back.

Edit: I'm using Windows 10, 64 bits, build 1709.

**Comment by bayleedev** [comment]
Yes, this is an annoying issue. I'm hoping electron can soon control the focused window!
